website/docs/docs/collaborate/govern/model-versions.md

```diff
@@ -5,10 +5,14 @@ sidebar_label: "Model versions"
 description: "Version models to help with lifecycle management"
 ---
 
+<VersionBlock lastVersion="1.5">
```

Comment:
I don't think we need this `VersionBlock` - the whole page is versioned >=1.5:

https://github.com/dbt-labs/docs.getdbt.com/blob/22ecf93c95dbd49c12b96c12bfd61c83fe7e58dd/website/dbt-versions.js#L42-L45

Reply:
thank you! there was a merge conflict earlier which i resolved but noticed this pr had version blocks and the existing page doesn't. appreciate the confirmation 🙏 !